packages/store/src/patterns/interface-tools.js

```diff
@@ -144,6 +144,7 @@ const bindMethod = (
   let { method } = thisfulMethods
     ? {
         method(...args) {
+          this || Fail`thisful method ${methodTag} called without this`;
```

Comment:
@erights, is this correct?

Reply:
It is not incorrect, but it is redundant. In the case where this emits a diagnostic, the following `getContext(this)` will already emit a similar diagnostic. I recommend dropping the test here and relying on `getContext` to emit good diagnostics. If the `getContext` diagnostic is not clear enough, better to improve it there.

Reply:
It's redundant, yes. This is to give a more helpful error message. I considered putting it in `getContext` but that's used also for non-thisful methods so I didn't see a way for it to give this counsel.

Regardless I'll remove the commit from this PR since it merits more discussion.